### DISCUSSION TITLE
useTooltipTrigger support  for custom exit delay

### DISCUSSION THREAD

**Discussion by Agile-fox** [adobe/react-spectrum#4205] [Ideas]
useTooltipTrigger hook is using `delay` prop to set a custom delay for a tooltip when it shows. It works great but don't offer a way to set up a custom exit delay for the tooltip when it disappear.

Small feature but I think could be handy for some specific scenarios.

**Comment by devongovett** [comment]
It was just added in #4128 and will be in the next release. For now, you're welcome to try the nightly version!

**Comment by Agile-fox** [comment]
Great news, thanks for the info 😄 I will try that soon.
